examples/backtest.ipynb

```diff
@@ -23,15 +23,33 @@
     "\n",
```

Comment:
<p>In this section we will analyze the backtest results from the different point of views. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tinkoff-ai/etna/pull/990/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>